graf3d/eve7/CMakeLists.txt

```diff
@@ -137,6 +137,20 @@ if(builtin_nlohmannjson)
    target_include_directories(ROOTEve PRIVATE ${CMAKE_SOURCE_DIR}/builtins)
 else()
    target_link_libraries(ROOTEve PUBLIC nlohmann_json::nlohmann_json)
+
+   get_target_property(inc_dirs nlohmann_json::nlohmann_json INTERFACE_INCLUDE_DIRECTORIES)
+   foreach(dir ${inc_dirs})
+      if(EXISTS "${dir}/nlohmann/json_fwd.hpp")
+         set(found_fwd true)
+      endif()
+   endforeach()
```

Comment:
Is it really worth it going to such great lengths just to avoid including the full header? I am beginning to think it's probably not.